scripts/load-climate-baseline-values.py

```diff
@@ -30,6 +29,12 @@
 
     f = open(args.file)
 
+    # Header processing: decduced from file and from R code that processes it
+    # Headers are optional
+    # Header if present, is 2 lines:
+    #   - crs?: GEO | ALB | UTM
+    #   - some mysterious number, e.g, 21
+    # We don't use these header values, and (naturally, therefore) we skip them if present
     line = next(f)
     if line.rstrip(' \0\n') in ['GEO','ALB','UTM']:
```

Comment:
This is getting pretty nit-picky, but usually with constants you want to use a tuple rather than a list. In this code, it won't really matter that much, but it could in higher performance, computationally intensive code.